cnf/make.conf.example

```diff
@@ -183,6 +183,34 @@
 # This ftp connection is active ftp.
 #PORTAGE_BINHOST="ftp://login:pass@grp.mirror.site:21*/pub/grp/i686/athlon-xp/"
 
+# Binary packages GPG commands
+# ============================
+#
+# Only works with GPKG format.
+# "binpkg-signing" needed to be set in FEATURES if need signing packages.
+# "binpkg-request-signature" needed to be set in FEATURES if you want all 
+# binpkgs must have signature.
+# You need uncomment related commands and set "USER" and "SIGN_KEY" to yours.
+#
+# Binary package GPG unlock command
+# Only for package signing.
+# You need this if you have to enter the GPG password manually AND/OR
+# You need to use gpg-keepalive feature to avoid GPG session expired.
+#BINPKG_GPG_UNLOCK_COMMAND="flock /run/lock/binpkg-gpg.lock sudo -u USER -- /usr/bin/gpg --detach-sig --armor --batch --yes --digest-algo SHA256 --local-user SIGN_KEY --output /dev/null /dev/null"
+#
+# Binary package GPG signing command
+# Only for package signing.
+# For security reasons, NOT recommended signing as root user.
+# Use flock to avoid GPG race condition.
+#BINPKG_GPG_SIGNING_COMMAND="flock /run/lock/binpkg-gpg.lock sudo -u USER -- /usr/bin/gpg --detach-sig --armor --batch --yes --no-tty --digest-algo SHA256 --local-user SIGN_KEY"
+#
+# Binary package GPG verify command
+# Only for package verification.
+# You can also use --homedir "PATH/TO/.gnupg" without sudo or non-standard path
+# \"\${SIGN_FILE}\" is needed and must be the first file to give as signature.
+# "-" must be the last parameter to tell GPG read data from stdin.
+#BINPKG_GPG_VERIFY_COMMAND="/usr/bin/sudo -u nobody -- /usr/bin/gpg --no-auto-check-trustdb --verify --batch --no-tty --homedir /etc/portage/gnupg --status-fd 1 \"\${SIGN_FILE}\" -"
```

Comment:
Is `nobody` really necessary or you meant `USER` as previously?

Reply:
Use `nobody` or not is up to users, I added that because https://lists.gnupg.org/pipermail/gnupg-announce/2021q1/000456.html, which seems it can be attacked even before the verification process is complete.

Reply:
Noted, I was not aware of this.